vignettes/SEIRD_age_structured.Rmd

```diff
@@ -78,6 +78,8 @@ c_all <- c_home %>%
 # plot all
 c_all %>%
   ggplot(aes(x=age_infector, y=age_infectee, fill=value)) +
+  xlab("age group of infector") + ylab("age group of infectee") +
+  theme(text = element_text(size=20)) +
```

Comment:
The text size is ok for the vertical axis; less so for the horizontal though (at least on my screen) since the text then overlaps. A potential idea is to rotate this text, but there may be easier solutions?